package.json

```diff
@@ -14,12 +18,20 @@
   "main": "background.js",
   "dependencies": {
     "@kubernetes/client-node": "^0.13.0",
+    "babel-plugin-module-resolver": "^4.0.0",
+    "bufferutil": "^4.0.3",
```

Comment:
I added those to deal with the warning messages at startup, so it would seem they aren't correctly identified as dependencies. And this search suggests that we need to explicitly include it:
```
$ grep -lr --include=package.json babel-plugin-module-resolver node_modules/
node_modules//babel-plugin-module-resolver/package.json
```

Reply:
This is what happens without the module-resolver:
```
vue-cli-service lint --no-fix

 ERROR  Error: Cannot find module 'babel-plugin-module-resolver'
Require stack:
- /Users/ericp/workspace/rancher/desktop/node_modules/@babel/core/lib/config/files/plugins.js
- /Users/ericp/workspace/rancher/desktop/node_modules/@babel/core/lib/config/files/index.js
- /Users/ericp/workspace/rancher/desktop/node_modules/@babel/core/lib/index.js
- /Users/ericp/workspace/rancher/desktop/node_modules/@vue/cli-plugin-babel/index.js
- /Users/ericp/workspace/rancher/desktop/node_modules/@vue/cli-service/lib/Service.js
- /Users/ericp/workspace/rancher/desktop/node_modules/@vue/cli-service/bin/vue-cli-service.js
Error: Cannot find module 'babel-plugin-module-resolver'
```

And, yes, I included `bufferutil` to deal with the warning message I saw at startup. I didn't bother searching for a "more correct" way of solving that.